### DISCUSSION TITLE
Post-translational Modifications in Martini3

### DISCUSSION THREAD

**Discussion by APSSMDS** [Martini-Force-Field-Initiative/MartiniDiscussions#3] [Proteins]
I am interested in simulating proteins with post-translational modifications of phosphorylated residues. In my current understanding, Martini3 does not currently include parameters for this type of PTM. Is my understanding correct and if so, are there any plans to expand Martini3 to include additional post-translational modification parameters?

**Comment by amiel-ab** [comment]
Hi @APSSMDS 
You can  have a look at this https://vermouth-martinize.readthedocs.io/en/latest/tutorials/6_adding_residues_links/index.html
Amiel
